Back-End/src/main/java/com/flight_search/mapper/FlightMapper.java

```diff
@@ -0,0 +1,14 @@
+package com.flight_search.mapper;
+
+import com.flight_search.dto.FlightDTO;
+import com.flight_search.model.Flight;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface FlightMapper {
+    FlightMapper INSTANCE = Mappers.getMapper(FlightMapper.class);
```

Comment:
why name is INSTANCE ?

Reply:
Stop using mappers and instead only use DTOs to parse the flight information.